pkg/acsengine/k8s_versions.go

```diff
@@ -8,6 +8,33 @@ import (
 
 // KubeConfigs represents Docker images used for Kubernetes components based on Kubernetes versions (major.minor.patch)
 var KubeConfigs = map[string]map[string]string{
+	common.KubernetesVersion1Dot9Beta1: {
+		"hyperkube": "hyperkube-amd64:v1.9.0-beta.1",
+		"ccm":       "cloud-controller-manager-amd64:v1.9.0",
```

Comment:
Here should be 'cloud-controller-manager-amd64:v1.9.0-beta.1'.

Another thing to note is azure's 'ccm' image release is planned to be shipped via public dockerhub, which does not share 'gcrio.azureedge.net' prefix then. This could be addressed later when we move to that image.